drivers/storage/portworx/deployment.go

```diff
@@ -49,13 +49,14 @@ const (
 )
 
 var (
-	pxVer2_3_2, _  = version.NewVersion("2.3.2")
-	pxVer2_5_5, _  = version.NewVersion("2.5.5")
-	pxVer2_6, _    = version.NewVersion("2.6")
-	pxVer2_8, _    = version.NewVersion("2.8")
-	pxVer2_9_1, _  = version.NewVersion("2.9.1")
-	pxVer2_13_8, _ = version.NewVersion("2.13.8")
-	pxVer3_0_1, _  = version.NewVersion("3.0.1")
+	pxVer2_3_2, _         = version.NewVersion("2.3.2")
+	pxVer2_5_5, _         = version.NewVersion("2.5.5")
+	pxVer2_6, _           = version.NewVersion("2.6")
+	pxVer2_8, _           = version.NewVersion("2.8")
+	pxVer2_9_1, _         = version.NewVersion("2.9.1")
+	pxVer2_13_8, _        = version.NewVersion("2.13.8")
+	pxVer3_0_1, _         = version.NewVersion("3.0.1")
+	supportedPxVersion, _ = version.NewVersion("2.13")
```

Comment:
`supportPXVersion` is too generic. What is supported is not clear.
Let's rename it to something like `csiRegistrarRemovalPxVersion`.